x-pack/test/functional/apps/api_keys/index.ts

```diff
@@ -0,0 +1,14 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { FtrProviderContext } from '../../ftr_provider_context';
+
+export default ({ loadTestFile }: FtrProviderContext) => {
+  describe('API Keys app', function() {
+    this.tags(['ciGroup7', 'skipCloud']);
```

Comment:
**question**: Why do we want to skip these tests on cloud?

Reply:
When this was created, I don't believe the app was on cloud yet. Removing skipcloud tag.